portmap/core/views.py

```diff
@@ -216,3 +219,54 @@ def debug_list_articles(request):
 
 def debug_help_dev(request):
     return TemplateResponse(request, "core/debug_index.html")
+
+class CorrectMimeTypeFeed(DefaultFeed):
+    content_type = 'application/xml; charset=utf-8'
+
+class RssFeed(Feed):
+    feed_type = CorrectMimeTypeFeed
+    title_template = "feed/article_title.html"
+    description_template = "feed/article_description.html"
+    title = "RSS Feed"
+    link = "https://portmap.dtinit.org/articles_feed"
+    description = "Articles from Portability Articles repo"
+
+    def items(self):
+        github_articles = "https://api.github.com/repos/dtinit/portability-articles/contents/articles"
+        response = requests.get(github_articles)
+        articles = response.json()
+
+        parsed_articles = []
+        for article in articles:
+            
+            name = article["name"]
+            html_url = article["html_url"]
+            content_response = requests.get(article['download_url'])
+
+            if content_response.status_code == 200:
+
+                markdown_content = content_response.text
```

Comment:
Transforming markdown to text should be a utility.  We already have something elsewhere that strips the YAML header from the content (and has had some of the bugs worked out of it) so we should re-use that.

FYI these are in portmap/utils.py which should be given a better name - really it's our MD/YAML handling.